code/modules/fish/fishtank.dm

```diff
@@ -322,8 +321,8 @@
 		to_chat(user, "<span class='notice'>[duds] egg\s [duds == 1 ? "was a dud" : "were duds"]!</span>")
 
 /obj/machinery/fishtank/proc/harvest_fish(mob/user)
-	if(fish_count <= 0)									//Can't catch non-existant fish!
-		to_chat(user, "There are no fish in [src] to catch!")
+	if(get_num_fish() <= 0)									//Can't catch non-existant fish!
+		to_chat(user, "<span class='warning'>There are no fish in [src] to catch!</span>")
 		return
 	var/list/fish_types = list()
 	var/list/fish_types_input = list()
```

Comment:
Can you add a comment here describing what this list looks like?
So describe what the key and value are. It took me a bit to understand what this list was used for and how it was made up. A comment could help a long way with that

Reply:
Done. Sorry for taking so long to get to this.